src/substrate/index.ts

```diff
@@ -26,7 +26,9 @@ async function handleOp(rid: string, op: CommandOp, args: any[]) {
     let name = eh[0];
     let handler = eh[1];
     if (handler.eventHandler) {
-      console.log("executing event handler ", name);
+      if(process.argv[2] == "v") {
```

Comment:
No need change. For normal mode, we are just avoiding those mass logs which can be increased when our request id become more and more, no need hide such static step logs.